kafka/instance.go

```diff
@@ -0,0 +1,111 @@
+package kafka
+
+import (
+	"context"
+
+	"github.com/IBM/sarama"
+)
+
+// Handler is a Kafka message function Handler, which is invoked when it
+// receives a Kafka message. It must implement the Handle method:
+//
+//	Handle(context.Context, *sarama.ConsumerMessage) error
+//
+// It can optionally implement any of Start, Stop, Ready, and Alive.
+type Handler interface {
+	// Handle a Kafka message.
+	Handle(context.Context, *sarama.ConsumerMessage) error
```

Comment:
I think I generally like more just deliver them as CloudEvents... I guess we could have also some native approach (e.g. like we have for http)